Collor/Classes/CollectionCellDescribable.swift

```diff
@@ -18,10 +18,21 @@ public protocol CollectionCellDescribable : Identifiable {
     var identifier: String { get }
     var className: String { get }
     var selectable: Bool { get }
-    func getAdapter() -> CollectionAdapter
-    func size(_ collectionView: UICollectionView, sectionDescriptor: CollectionSectionDescribable) -> CGSize
+    var adapter: CollectionAdapter { get }
```

Comment:
`func getAdapter()` instead of `var adapter` was created to be able to declare a typed property `adapter` in the descriptor class. With your new code, the adapter is no more typed. It's a problem, each we need to access to this adapter, we have to cast it.